hack/gen-swagger-doc/deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/bash
+
+set -e
+
+GITHUB_FQDN=github.com
+API_REF_REPO=kubevirt-incubator/api-reference
+API_REF_DIR=/tmp/api-reference
+
+git clone \
+    "https://${API_REFERENCE_PUSH_TOKEN}@${GITHUB_FQDN}/${API_REF_REPO}.git" \
+    "${API_REF_DIR}" > /dev/null 2>&1
+rm -rf "${API_REF_DIR}/content/"*
+cp -f hack/gen-swagger-doc/html5/content/*.html "${API_REF_DIR}/content/"
```

Comment:
we need a gitignore entry for that directory

Reply:
we have it there `hack/gen-swagger-doc/html5`